examples/models/openai_compatibility_model_examples/model_manger.py

```diff
@@ -0,0 +1,55 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+
+from camel.agents import ChatAgent
+from camel.models import ModelFactory
+from camel.types import ModelPlatformType
+
+# Use two different models for ModelManager
+
+model1 = ModelFactory.create(
+    model_platform=ModelPlatformType.OPENAI_COMPATIBLE_MODEL,
+    model_type="grok-beta",
+    api_key="xai-...",
+    url="https://api.x.ai/v1",
+    model_config_dict={"max_tokens": 2000},
+)
+
+model2 = ModelFactory.create(
+    model_platform=ModelPlatformType.OPENAI_COMPATIBLE_MODEL,
+    model_type="nvidia/llama-3.1-nemotron-70b-instruct",
+    api_key="nvapi-xx",
+    url="https://integrate.api.nvidia.com/v1",
+    model_config_dict={"temperature": 0.4},
+)
+
+assistant_sys_msg = "Testing two models in ModelManager"
+
+agent = ChatAgent(assistant_sys_msg, model=[model1, model2])
```

Comment:
I believe that we should expose the capability for the users to change the schedule strategy, please update and include an example here to show how to change the strategy. 
We could define an enum type here(https://github.com/camel-ai/camel/blob/master/camel/types/enums.py) and map the enum type to the strategy in the model manager. In the chatagent, expose a param for the user to set the schedule strategy.

Reply:
Extended an example. But I would avoid using Enum for scheduling_strategy param, as we allow user to use his own strategy implementation with own name.